### ISSUE TITLE
2x2 apartment tower has useless solid rock inside it

### ISSUE THREAD

**Issue by NetSysFire** [CleverRaven/Cataclysm-DDA#58162]
### Describe the bug

The 2x2 apartment tower has, regardless of wall type, some unfitting solid rock inside it.

![image](https://user-images.githubusercontent.com/59517351/172026872-06e0086d-c581-4f73-80c0-c374c0cab6b1.png)

### Steps to reproduce

https://github.com/CleverRaven/Cataclysm-DDA/blob/00766c9cd6eec92db42a6d238f54d528985c63c1/data/json/mapgen/apartment_con_new.json#L35-L36

### Expected behavior

Either repurpose it as a storage room storing junk such as cleaning utensils (I assume a janitor is in charge of such a large building) or, which can prove problematic, to replace the solid rock with walls (looks weird tbh) or support columns (afaik we only have stone/concrete and wood, no brick).

I am in favor of option 1, but if you can think of any other good solutions, feel free to implement these instead. The slight problem there is though that every floor has the solid rock, not just the ground level one. The "storing junk" excuse only works for the ground level floor because one of these spaces is only connected to the elevator and can thus only be reached from the outside.

All the changes are easy to make and are appropriate for new contributors. The real challenge is to actually find out what fits. Labelling as good first issue.

### Versions and configuration

master

**Comment by hexagonrecursion** [comment]
I suggest we make two apartments on each floor a bit bigger. Seems the most logical solution to me: more square meters = more expensive apartment.

**Comment by GuardianDll** [comment]
is this a problem? i mean, its pretty hard to use this 2 tile in the game, and they are a pretty common thing in the game, with our 1_meter_width walls due to the game engine. But i can understand the same pr about [the hospital)](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/mapgen/hospital.json#L57) map, that has a huge 10x6 empty space, filled with walls, just because no one know what to put here, and no one loot it really often.

**Comment by NetSysFire** [comment]
>is this a problem?

Yes, albeit not a critical one. It should be improved if possible, just like the space in the hospital. Would be nice if someone found a solution for both.

**Comment by NetSysFire** [comment]
![image](https://user-images.githubusercontent.com/59517351/176591691-3b7c708a-7b38-4ed8-a9b3-2805c6926a14.png)

This is the office tower.
